sources/packages/api/src/app.aest.module.ts

```diff
@@ -92,6 +95,10 @@ import { ApplicationControllerService } from "./route-controllers/application/ap
     GCNotifyService,
     GCNotifyActionsService,
     AssessmentControllerService,
+    StudentControllerService,
```

Comment:
The StudentControllerService has one dependency i.e StudentFileService. Why are StudentService,
    SFASIndividualService added to module?

Reply:
`StudentService` requires `SFASIndividualService`.